lib/IRGen/DebugTypeInfo.h

```diff
@@ -72,8 +72,9 @@ class DebugTypeInfo {
   static DebugTypeInfo getTypeMetadata(swift::Type Ty, llvm::Type *StorageTy,
                                        Size size, Alignment align);
 
-  /// Create a forward declaration for a type whose size is unknown.
-  static DebugTypeInfo getForwardDecl(swift::Type Ty);
+  /// Create a forward declaration for a type whose size is unknown (forward
+  /// decls, unsubstituted generic type parameters)
+  static DebugTypeInfo getFromType(swift::Type Ty);
```

Comment:
If this still created a forward declaration, I'd rather keep that in the function name?